features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/apis/v1/entitlement_api_router.jag

```diff
@@ -15,6 +15,7 @@ require('/modules/publisher.js').exec(function(ctx) {
 
     routeManager.register('GET', 'publisher', '/publisher/api/entitlement/get/Tiers', function (context) {
         var tiers = getTiers();
+        response.contentType = "application/json";
```

Comment:
fixed for this file & others in commit cd872e95c29ac3f1e1c5b882c1e28cdb5a8eab81 of PR #240